pkg/metadata/sandbox.go

```diff
@@ -55,7 +67,9 @@ type SandboxMetadata struct {
 	// CreatedAt is the created timestamp.
 	CreatedAt int64
 	// NetNS is the network namespace used by the sandbox.
-	NetNS string
+	NetNS *SandboxNetNS
```

Comment:
The PR is out. https://github.com/kubernetes-incubator/cri-containerd/pull/66
`NetNS` should be in https://github.com/kubernetes-incubator/cri-containerd/pull/66/files#diff-e723511e95258721117711bb80b67136R27.

I'll try to get it merged as soon as possible.